### DISCUSSION TITLE
Persistent Networkmanager Notification

### DISCUSSION THREAD

**Discussion by AlphaElwedritsch** [SpiralLinux/SpiralLinux-project#353] [Q&A] [answered]
SpiralLinux - Budie Desktop

Every time I connect to my WiFi network I get the notification.
No matter whether after rebooting and logging in, or when the laptop wakes up from standby.
The "Don't show this message again" function has no effect.
The message keeps appearing.

Side note:
I also had the same problem under Mint LMDE with cinnamon.

Is there a way to permanently suppress this message?

Thanks and regards
alpha

<a href="https://imgbb.com/"><img src="https://i.ibb.co/JkgHMNK/Snapshot-2023-11-14-07-24-24.png" alt="Snapshot-2023-11-14-07-24-24" border="0"></a>

**Comment by AlphaElwedritsch** [comment]
Can nobody help?
To not getting this notification again and again, every time laptop is reconnection to WiFi network.

**Comment by AlphaElwedritsch** [comment, marked as answer]
Will try these both commands in the evening


gsettings set org.gnome.nm-applet disable-disconnected-notifications "true" 
gsettings set org.gnome.nm-applet disable-connected-notifications "true" 

See what happens

**Reply by AlphaElwedritsch** [reply]
It's doing the job

**Reply by geckolinux** [reply]
Glad you figured it out. You can also use the pre-installed Dconf Editor to change those settings in a visual way.
